### DISCUSSION TITLE
Errors while building ImageMagick with VS Community 2013

### DISCUSSION THREAD

**Discussion by avin15cs** [ImageMagick/ImageMagick#3115] [Development]
I am using VS Community 2013 to build ImageMagick. I have downloaded ImageMagick-windows.zip(125M) file from [here](https://download.imagemagick.org/ImageMagick/download/windows/).

OS: Windows Server 2012 R2

Steps performed:

1. Downloaded ImageMagick-windows.zip from http://www.imagemagick.org/download/windows.
2. Opened configure.sln from \ImageMagick-7.X.X\VisualMagick\configure with VS Community 2013 and Built the solution in release mode.
3. Ran Configure.exe from \ImageMagick-7.X.X\VisualMagick\configure. Clicked Next-> Selected Static Multi-Threaded DLL runtimes -> next-> next -> Finish.
4. Opened VisualStaticMTD.sln and Then Built solution in release mode.

This has resulted me with 16k+ errors.

On following the same steps with VS 2019 it works fine and convert.exe is generated, but I need it with VS 2013.

Have I missed adding any dependencies? Can anyone please help me on this?

**Comment by dlemstra** [comment]
We haven't tested this with older versions of Visual Studio for a while an we should probably remove those options from the configure program. It is very likely that one of the libraries that we use uses something that is only compatible with recent version(s) of Visual Studio.

**Reply by avin15cs** [reply]
Hi @dlemstra ,
So, Wont I be able to Build it in VS 2013? But, I saw that the solution was built using VS120.

**Reply by dlemstra** [reply]
You might be able to do that. But you will need to figure that out for yourself. It could be possible that you need to remove some new formats to make it possible to build the solution.
